box/info.go

```diff
@@ -0,0 +1,100 @@
+package box
+
+import (
+	"fmt"
+
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+var _ tarantool.Request = (*InfoRequest)(nil)
+
+// ClusterInfo represents information about the cluster.
+// It contains the unique identifier (UUID) of the cluster.
+type ClusterInfo struct {
+	UUID string `msgpack:"uuid"`
+}
+
+// Info represents detailed information about the Tarantool instance.
+// It includes version, node ID, read-only status, process ID, cluster information, and more.
+type Info struct {
+	// The Version of the Tarantool instance.
+	Version string `msgpack:"version"`
+	// The node ID (nullable).
+	ID *int `msgpack:"id"`
+	// Read-only (RO) status of the instance.
+	RO bool `msgpack:"ro"`
+	// UUID - Unique identifier of the instance.
+	UUID string `msgpack:"uuid"`
+	// Process ID of the instance.
+	PID int `msgpack:"pid"`
+	// Status - Current status of the instance (e.g., running, unconfigured).
+	Status string `msgpack:"status"`
+	// LSN - Log sequence number of the instance.
+	LSN uint64 `msgpack:"lsn"`
+}
+
+// InfoResponse represents the response structure
+// that holds the information of the Tarantool instance.
+// It contains a single field: Info, which holds the instance details (version, UUID, PID, etc.).
+type InfoResponse struct {
+	Info Info
+}
+
+func (ir *InfoResponse) DecodeMsgpack(d *msgpack.Decoder) error {
+	arrayLen, err := d.DecodeArrayLen()
+	if err != nil {
+		return err
+	}
+
+	if arrayLen != 1 {
+		return fmt.Errorf("protocol violation; expected 1 array entry, got %d", arrayLen)
+	}
+
+	i := Info{}
+	err = d.Decode(&i)
+	if err != nil {
+		return err
+	}
+
+	ir.Info = i
+
+	return nil
+}
+
+// Info retrieves the current information of the Tarantool instance.
+// It calls the "box.info" function and parses the result into the Info structure.
+func (b *box) Info() (Info, error) {
+	var infoResp InfoResponse
+
+	// Call "box.info" to get instance information from Tarantool.
+	fut := b.conn.Do(NewInfoRequest())
+
+	// Parse the result into the Info structure.
+	err := fut.GetTyped(&infoResp)
+	if err != nil {
+		return Info{}, err
+	}
+
+	// Return the parsed info and any potential error.
+	return infoResp.Info, err
+}
+
```

Comment:
Please, move it to `box.go`. The request/response is ok here.

Reply:
done